modules/reissue-internal-certificates-central.adoc

```diff
@@ -7,21 +7,31 @@
 
 Central uses a built-in server certificate for authentication when communicating with other {product-title} services.
 This certificate is unique to your Central installation.
-The RHACS portal shows an information banner when the Central certificate is about to expire.
+The {product-title-short} portal shows an information banner when the Central certificate is about to expire.
 
 [NOTE]
 ====
-The information banner only appears 15 days before the certificate expiry date.
+The information banner only appears 15 days before the certificate expiration date.
 ====
 
+For {ocp} installations, beginning with {product-title-short} version 4.3.4, the Operator will automatically rotate all Central components' service TLS certificates 6 months before they expire. The following conditions apply:
```

Comment:
I thinks this applies not only for OCP, but for RHACS operator installations in general.

AFAIK the operator is only officially supported for OCP, so as far as official support goes this statement is true.